diff --git a/package/libs/openssl/Makefile b/package/libs/openssl/Makefile
index 3d13fe0..8579de5 100644
--- a/package/libs/openssl/Makefile
+++ b/package/libs/openssl/Makefile
@@ -20,7 +20,8 @@ PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://www.openssl.org/source/ \
 	http://www.openssl.org/source/old/$(PKG_BASE)/ \
 	ftp://ftp.funet.fi/pub/crypt/mirrors/ftp.openssl.org/source \
-	ftp://ftp.sunet.se/pub/security/tools/net/openssl/source/
+	ftp://ftp.sunet.se/pub/security/tools/net/openssl/source/ \
+	http://ftp.vim.org/security/openssl/
 PKG_MD5SUM:=9392e65072ce4b614c1392eefc1f23d0
 
 PKG_LICENSE:=OpenSSL
